From 62aabc57e6c18798a8232a7a560c35ebfe088d91 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Mon, 4 Oct 2021 17:14:34 +0200
Subject: [PATCH] Remove problematic IgnOGRE logic

---
 CMakeLists.txt | 28 ++++------------------------
 1 file changed, 4 insertions(+), 24 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 51328f1e..719f7721 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -64,30 +64,10 @@ endif()
 # Find OGRE
 list(APPEND ign_ogre_components "RTShaderSystem" "Terrain" "Overlay")
 
-# Ogre versions greater than 1.9 are not officialy supported.
-# Display a warning for the users on this setup unless they provide
-# USE_UNOFFICIAL_OGRE_VERSIONS flag
-if (NOT USE_UNOFFICIAL_OGRE_VERSIONS)
-  # Only for checking the ogre version
-  ign_find_package(IgnOGRE VERSION 1.10 QUIET)
-
-  if (OGRE_FOUND)
-    IGN_BUILD_WARNING("Ogre 1.x versions greater than 1.9 are not officially supported."
-                      "The software might compile and even work but support from upstream"
-                      "could be reduced to accepting patches for newer versions")
-    ign_find_package(IgnOGRE VERSION 1.10
-      COMPONENTS ${ign_ogre_components}
-      REQUIRED_BY ogre
-      PRIVATE_FOR ogre)
-  else()
-    # If ogre 1.10 or greater was not found, then proceed to look for 1.9.x
-    # versions which are offically supported
-    ign_find_package(IgnOGRE VERSION 1.9.0
-      COMPONENTS ${ign_ogre_components}
-      REQUIRED_BY ogre
-      PRIVATE_FOR ogre)
-  endif()
-endif()
+ign_find_package(IgnOGRE VERSION 1.10
+  COMPONENTS ${ign_ogre_components}
+  REQUIRED_BY ogre
+  PRIVATE_FOR ogre)
 
 if (OGRE_FOUND)
   # find boost - mainly needed on macOS and also by the terrain component
